packages/cxl-ui/package.json

```diff
@@ -18,12 +18,14 @@
     "@vaadin/vaadin-accordion": "^2.0.0-alpha5",
     "@vaadin/vaadin-button": "^3.0.0-alpha1",
     "@vaadin/vaadin-context-menu": "^5.0.0-alpha2",
+    "@vaadin/vaadin-dialog": "^2.5.2",
     "@vaadin/vaadin-progress-bar": "^2.0.0-alpha2",
     "@vaadin/vaadin-tabs": "^4.0.0-alpha4",
     "@vaadin/vaadin-themable-mixin": "^1.6.2",
     "headroom.js": "^0.12.0",
     "lit-element": "^2.2.1",
-    "lit-html": "^1.1.2"
+    "lit-html": "^1.1.2",
+    "lorem-ipsum": "^2.0.3"
```

Comment:
Should `lorem-ipsum` be a `storybook` dependency, instead?

Reply:
That makes sense.